sonarapp/Program.cs

```diff
@@ -13,6 +13,9 @@ static void Main(string[] args)
             string test = "This is a test";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'test' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)